plcx/utils/coroutine.py

```diff
@@ -0,0 +1,10 @@
+import inspect
+from typing import Any
+
+
+async def await_if_coroutine(obj: Any, *args, **kwargs) -> Any:
```

Comment:
If we want the type to be correct, `obj` should be typed as `Callable`, not `Any`.